### ISSUE TITLE
Getting error when openinng the app / project on VSCode

### ISSUE THREAD

**Issue by terealcantara** [wallabyjs/public#1299]
### Issue description or question

##getting this error since yesterday 9/11/2017:
[WallabyJs.wallaby-vscode]: Proposed API is only available when running out of dev or with the following command line switch: --enable-proposed-api WallabyJs.wallaby-vscode

##and getting error when starting wallaby: Please open some folder/project before starting wallaby.js.

### Wallaby.js configuration file

```javascript
var wallabyWebpack = require('wallaby-webpack');
var path = require('path');

var compilerOptions = Object.assign(
  require('./tsconfig.json').compilerOptions,
  require('./src/tsconfig.spec.json').compilerOptions);

module.exports = function (wallaby) {

  var webpackPostprocessor = wallabyWebpack({
    entryPatterns: [
      'src/wallabyTest.js',
      'src/**/*spec.js'
    ],

    module: {
      loaders: [
        {test: /\.css$/, loader: 'raw-loader'},
        {test: /\.html$/, loader: 'raw-loader'},
        {test: /\.js$/, loader: 'angular2-template-loader', exclude: /node_modules/},
        {test: /\.json$/, loader: 'json-loader'},
        {test: /\.styl$/, loaders: ['raw-loader', 'stylus-loader']},
        {test: /\.less$/, loaders: ['raw-loader', 'less-loader']},
        {test: /\.scss$|\.sass$/, loaders: ['raw-loader', 'sass-loader']},
        {test: /\.(jpg|png)$/, loader: 'url-loader?limit=128000'}
      ]
    },

    resolve: {
      modules: [
        path.join(wallaby.projectCacheDir, 'src/app'),
        path.join(wallaby.projectCacheDir, 'src')
      ]
    }
  });

  return {
    files: [
      {pattern: 'src/**/*.ts', load: false},
      {pattern: 'src/**/*.d.ts', ignore: true},
      {pattern: 'src/**/*.css', load: false},
      {pattern: 'src/**/*.less', load: false},
      {pattern: 'src/**/*.scss', load: false},
      {pattern: 'src/**/*.sass', load: false},
      {pattern: 'src/**/*.styl', load: false},
      {pattern: 'src/**/*.html', load: false},
      {pattern: 'src/**/*.json', load: false},
      {pattern: 'src/**/*spec.ts', ignore: true}
    ],

    tests: [
      {pattern: 'src/**/*spec.ts', load: false}
    ],

    testFramework: 'jasmine',

    compilers: {
      '**/*.ts': wallaby.compilers.typeScript(compilerOptions)
    },
    
    middleware: function (app, express) {
      var path = require('path');
      app.use('/favicon.ico', express.static(path.join(__dirname, 'src/favicon.ico')));
      app.use('/assets', express.static(path.join(__dirname, 'src/assets')));
    },

    env: {
      kind: 'electron'
    },

    postprocessor: webpackPostprocessor,

    setup: function () {
      window.__moduleBundler.loadTests();
    },

    debug: true
  };
};
```

### Code editor or IDE name and version

WebStorm v?
IntelliJ IDEA v?
Visual Studio v?
Atom v1.?
Visual Studio Code v1.? yes Version 1.14.0 (1.14.0)
Sublime Text v3 build ?

### OS name and version

Windows
OSX yes Sierra 10.12.6
Linux

**Comment by ArtemGovorov** [comment]
Thanks for the report, will be looking into it shortly. As a workaround, I may suggest upgrading your VS Code to 1.15.

**Comment by ArtemGovorov** [comment]
The backward compatibility issue is fixed for VS Code 1.14 and the fixed is published in v1.0.59.
